functions-and-operators/tidb-functions.md

```diff
@@ -230,3 +231,69 @@ See also:
 
 - [`Statement Summary Tables`](/statement-summary-tables.md)
 - [`INFORMATION_SCHEMA.TIDB_TRX`](/information-schema/information-schema-tidb-trx.md)
+
+### TIDB_SHARD
+
+The `TIDB_SHARD` function can be used to create a shard index to scatter the index hotspot. A shard index is an expression index prefixed with a `TIDB_SHARD` function.
+
+#### Shard index
+
+- Creation:
+
+    You can use `uk((tidb_shard(a)), a))` to create a shard index for the field `a`. When there is a hotspot caused by monotonically increasing or decreasing data on the index field `a` in the unique secondary index `uk((tidb_shard(a)), a))`, the index's prefix `tidb_shard(a)` can scatter the hotspot to improve the cluster's scalability.
+
+- Scenarios:
+
+    - When there is a write hotspot caused by monotonically increasing or decreasing keys on the unique secondary index, and the index contains integer type fields.
+    - The SQL statement executes an equality query based on all fields of the secondary index, either as a separate `SELECT` or as an internal query generated by `UPDATE`, `DELETE` and so on. The equality query includes two ways: `a = 1` or `a IN (1, 2, ......)`.
+
+- Limitations:
+
+    - Shard index cannot be used in inequality queries.
+    - Shard index cannot be used in queries that contain `OR` mixed with an outmost `AND` operator.
+    - Shard index cannot be used in the `GROUP BY` clause.
+    - Shard index cannot be used in the `ORDER BY` clause.
+    - Shard index cannot be used in the `ON` clause.
+    - Shard index cannot be used in the `WHERE` subquery.
+    - Shard index can only scatter unique indexes of integer fields.
+    - Shard index might not take effect in composite indexes.
+    - Shard index cannot go through FastPlan process, which affects optimizer performance.
+    - Shard index cannot be used in preparing the execution plan cache.
```

Comment:
Because all items start with "Shard index", it is suggested to remove the common subject.

Reply:
Cannot be used to ...